fragdenstaat_de/fds_blog/templates/fds_blog/_article_detail.html

```diff
@@ -20,12 +20,14 @@ <h1 class="blog-header__title">
                         {% with authors=article.get_authors %}
                             {% if authors %}
                                 <span> -
+                                    {# djlint:off #}
```

Comment:
Could [`spaceless`](https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#spaceless) work as well here, so we don't have to disable djlint?

Reply:
@pajowu Unfortunately not, as `spaceless` only removes whitespace between tags, but not within them.

Reply:
Sad